project/generate-dockerfile.sh

```diff
@@ -0,0 +1,255 @@
+#!/bin/bash
+set -e
+
+cat <<-'EOF'
+# This file describes the standard way to build Docker, using docker
```

Comment:
@brahmaroutu missing sentence saying `# DO NOT EDIT - GENERATED BY project/generate-dockerfile.sh`